packages/psss/src/components/Table/CellComponents/ArchivedAlertMenuCell.js

```diff
@@ -1,9 +1,9 @@
 import React, { useCallback, useContext } from 'react';
+import PropTypes from 'prop-types';
 import { ActionsMenu } from '@tupaia/ui-components';
-import { ArchiveTableContext } from '../../../containers/Tables';
 import * as COLORS from '../../../constants/colors';
 
-export const ArchivedAlertMenuCell = React.memo(({ id }) => {
+export const ArchivedAlertMenuCell = React.memo(({ id, ArchiveTableContext }) => {
```

Comment:
I haven't seen this pattern again (passing the context as prop), instinctively it seems a bit unexpected but it may be fine - would like to get another opinion from @tcaiger 

I assume that the motivation for this change was to avoid an import dependency cycle, and I know that this is sometimes tricky to achieve, so I'm personally not fussed either way!

Reply:
Or could we extract `ArchiveTableContext` in a new file and thus avoid the dependency loop?

Reply:
@chris-bes could you follow up on this one?

Reply:
Yes, I agree with Kostas. thanks

Reply:
Definitely was a poor effort to fix the dependency cycle issue 🤦‍♂️ . Thanks for pointing that out @kael89 !